code/src/App.js

```diff
@@ -1,9 +1,102 @@
-import React from 'react'
+import React, { useState, useEffect } from "react";
+
+import { FormInput } from "./components/FormInput";
+import { MessageList } from "./components/MessageList";
+import { LoadingSpinner } from "./components/LoadingSpinner";
+import { Animation } from "./components/Animation";
+
+import { API_URL, LIKES_URL } from './reusables/urls';
 
 export const App = () => {
+  const [newMessage, setNewMessage] = useState("");
+  const [messageList, setMessageList] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [animate, setAnimation] = useState(false);
+  const [errorMessage, setErrorMessage] = useState("");
+  
+  useEffect(() => {
+    fetchAllMessages();
+  }, []);
+
+  // Fetching all the thoughts
+  const fetchAllMessages = () => {
+    fetch(API_URL)
+      .then(res => res.json())
+      .then(messages => {
+        setMessageList(messages);
+        setLoading(false);
+        setAnimation(false);
+      })
+    .catch(err => console.error(err));  
```

Comment:
console.log cant be present in the production code.

Reply:
Removed them now! Didn't think about the ones in the catch :)